.github/workflows/main.yml

```diff
@@ -0,0 +1,146 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Docker Build
+
+# Controls when the action will run. 
+on:
+  release: 
+    types: [published]
+
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    env:
+      # Setting the defaults up front
+      LATEST_NODE: 14
+      DEFAULT_IMAGE: hardillb/testing
```

Comment:
hardillb/testing  ?

Reply:
Doh, that's my test docker repo, will update later